src/workflow/tasks.py

```
@@ -130,7 +175,10 @@ def execute_workflow(
                 log_file = log_dir / log_name
                 with open(log_file, "a") as f:
                     f.write(f"\n\nERROR: {str(e)}\n")
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)